.codeclimate.yml

```diff
@@ -3,5 +3,6 @@ languages:
   PHP: true
 exclude_paths:
 - "lib/strip_attributes/test/*"
+- "db/migrate/"
```

Comment:
Added to prevent CodeClimate complaining about duplicated code in database migrations

Reply:
Commit messages are the place for this kind of comment ☝️ 

If in a years time I'm wondering why we excluded `"db/migrate/", first thing I'd do is git blame on the line. Takes time to then trace that back to a PR (which are more ephemeral than commits – e.g. if we decided to move to GitLab or something).